component/storage/mongodb/mongodb.go

```diff
@@ -0,0 +1,280 @@
+/*
+Copyright Scoir Inc Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package mongodb implements a storage interface for Aries (aries-framework-go).
+//
+package mongodb
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// Data is a wrapper for the stored key/value pair.
+type Data struct {
```

Comment:
Do we need to export this struct ? Looks like this struct is only for internal use.

Reply:
Local makes more sense.  Fixed.